src/Setup/DevDivVsix/MicrosoftCodeAnalysisExpressionEvaluator/Microsoft.CodeAnalysis.ExpressionEvaluator.Resources.swr

```diff
@@ -0,0 +1,7 @@
+use vs
+
+package name=Microsoft.CodeAnalysis.ExpressionEvaluator.Resources
+        version=$(Version)
+        vs.package.language=$(LangCountry)
+
+# No files to work around bug #239676
```

Comment:
Please include a URL to the issue. Issue numbers by themselves aren't sufficient for tracking down bugs.